examples/rl/cim/ac.py

```diff
@@ -11,7 +11,7 @@
 from maro.rl.model import DiscreteACNet, FullyConnectedBlock, OptimOption
 from maro.rl.policy.algorithms import ActorCritic, ActorCriticConfig
 
-cim_path = os.path.dirname(os.path.dirname(__file__))
+cim_path = os.path.dirname(os.path.realpath(__file__))
 sys.path.insert(0, cim_path)
```

Comment:
Is there any special purpose for inserting `cim_path` into the first position of `sys.path`?

Reply:
Need to import from files under the cim dir